wallet/lib/walletService.ts

```diff
@@ -0,0 +1,51 @@
+// so we need the http client to make a request to the wallet service
+// additionally the wallet service probably is protected by the JWT token so this needs to be passed as well
+
+
+export const DefaultWalletServiceBaseURL = "http://localhost:8081"
```

Comment:
where is this configured externally?

Reply:
this is configured in constructor here https://github.com/iotaledger/MPEG-21-Part-23-DLT-Manager/pull/2/files#diff-d671f4fa983aef3f2b3535e4ee97b908251541597486c6756e2e40e51226de56R27  So the assumption is the new web app will be used it as library, and this is up to web application configure the proper endpoint